internal/cmd/iam/command_rbac_role_binding_create.go

```diff
@@ -26,7 +26,7 @@ func (c *roleBindingCommand) newCreateCommand() *cobra.Command {
 			},
 			examples.Example{
 				Text: `Grant the role "ResourceOwner" to the principal "User:u-123456", in the environment "env-12345" for the Kafka cluster "lkc-123456" on the resource "Topic:my-topic":`,
-				Code: "confluent iam rbac role-binding create --principal User:u-123456 --role ResourceOwner --resource Topic:my-topic --environment env-12345 --kafka-cluster lkc-123456",
+				Code: "confluent iam rbac role-binding create --principal User:u-123456 --role ResourceOwner --resource Topic:my-topic --environment env-12345 --cloud-cluster lkc-123456 --kafka-cluster lkc-123456",
```

Comment:
This is technically not needed. It's inferred from the `--kafka-cluster` flag.

Reply:
@brianstrauch Thanks for taking a look @brianstrauch. A customer and Support rep mentioned stated that the examples currently don't. Can you take a look at the customer/support discussion in these two tickets: 

- https://confluentinc.atlassian.net/browse/DOCS-18192
- Support's last comment here: https://confluent.zendesk.com/agent/tickets/123340

Reply:
Responded in the DOCS ticket. The customer might be running a very old version of the CLI, and they're encountering a separate issue.

Reply:
Okay cool. Thanks @brianstrauch. 👍